src/Language/DslTransformer.php

```
@@ -0,0 +1,165 @@
+                    ];
+                    $relations = [];
+                }
+                $currentType = $m[1];
+                continue;
+            }
+
+            if ($line === 'relations') {
+                continue;
+            }
+
+            if (preg_match('/^define\s+(\w+)\s*:\s*(.+)$/', $line, $m)) {
```

PHPStan (2.1.15)

Only booleans are allowed in an if condition, int|false given.


(if.condNotBoolean)